lib/sqlalchemy/dialects/postgresql/array.py

```diff
@@ -322,81 +354,87 @@ def overlap(self, other):
     comparator_factory = Comparator
 
     @property
-    def hashable(self):
+    def hashable(self) -> bool:  # type: ignore[override]
         return self.as_tuple
 
     @property
-    def python_type(self):
+    def python_type(self) -> Type[Any_]:
         return list
 
-    def compare_values(self, x, y):
-        return x == y
+    def compare_values(self, x: Any_, y: Any_) -> bool:
+        return x == y  # type: ignore[no-any-return]
```

Comment:
we could probably just remove hashable, python_type and compare_values since they are the same as the base type (just a consideration)